sdk/test/test_objectstore.py

```diff
@@ -65,11 +65,11 @@ def setUp(self) -> None:
         )
 
     def test_store_retrieve(self) -> None:
-        object_store: ObjectStore[AssetAdministrationShell] = ObjectStore()
+        object_store: ObjectStore[Identifiable] = ObjectStore()
         object_store.add(self.aas1)
         object_store.add(self.aas2)
         self.assertIn(self.aas1, object_store)
-        property = aas_types.Property('test')
+        property = aas_types.Property('test')  # type: ignore
```

Comment:
Same here: 

```Python
# (2024-09-12, somsonson)
# We tell mypy to ignore this type because of <reason>
```

That way, in the future, we don't have to guess why we do not care (or rather cannot test) type safety at this point.